usaspending_api/references/management/commands/load_object_classes.py

```diff
@@ -0,0 +1,227 @@
+"""
+As of this writing, the raw source object class file can be found on S3 at s3://gtas-sf133/object_class.csv
```

Comment:
Is this file publicly accessible?

If yes, use the public URL. If not, then remove